lib/std/special/compiler_rt/stack_probe.zig

```diff
@@ -118,6 +105,21 @@ fn win_probe_stack_only() void {
         },
         else => {},
     }
+    if (comptime native_arch.isAARCH64()) {
+        // NOTE: page size hardcoded to 4096 for now
+        asm volatile (
+            \\        lsl    x16, x15, #4
+            \\        mov    x17, sp
+            \\1:
+            \\
+            \\        sub    x17, x17, 4096
```

Comment:
Note, I'm not sure how to pass `std.mem.page_size` here for the immediate operand to this instruction and the next one.